### PR TITLE
Update simorgh to set environment vars

### PR DESCRIPTION
**Pull request by nilchia**
Simorgh is extremely slow on EU. The reason is that torch sees all available cores in the node and tries to split the computation between them, however the job has less allocated cores. That causes the tool to be extremely slow.

So although in TPV the cores is set to 5 (or 20), but torch can see all available ( I don't know how)

These env variables will restrict that.

FOR CONTRIBUTOR:
* [x] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] License permits unrestricted use (educational + commercial)
* [ ] This PR adds a new tool or tool collection
* [x] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

There are two labels that allow to ignore specific (false positive) tool linter errors:

* `skip-version-check`: Use it if only a subset of the tools has been updated in a suite.
* `skip-url-check`: Use it if github CI sees 403 errors, but the URLs work.

### CODE REVIEW

**Comment by nilchia** [comment]
FYI:
```
nilchia@bilp69:~$ docker run --cpus 2 -it ghcr.io/amisteromid/simorgh:v1.0.1 bash
root@af4fe8b94b7e:/opt/SIMORGH# python
Python 3.11.9 | packaged by conda-forge | (main, Apr 19 2024, 18:36:13) [GCC 12.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import torch, os
>>> print('nproc/cpu_count       :', os.cpu_count())
nproc/cpu_count       : 12
>>> print('torch default threads :', torch.get_num_threads())
torch default threads : 10
>>> 


```
```
nilchia@bilp69:~$ docker run --cpus 2 -e OMP_NUM_THREADS=2 -e MKL_NUM_THREADS=2 -it ghcr.io/amisteromid/simorgh:v1.0.1 bash
root@05ee59c94078:/opt/SIMORGH# python
Python 3.11.9 | packaged by conda-forge | (main, Apr 19 2024, 18:36:13) [GCC 12.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import torch, os
>>> print('nproc/cpu_count       :', os.cpu_count())
nproc/cpu_count       : 12
>>> print('torch default threads :', torch.get_num_threads())
torch default threads : 2
>>> 


```

**Comment by bgruening** [comment]
Keep this open for any further update

**Review by bgruening — APPROVED** [review]
